### PR TITLE
CIV-11304 Order Made

### PR DESCRIPTION
**Pull request by miguelMolina3691**
### JIRA link (if applicable) ###

[CIV-11304](https://tools.hmcts.net/jira/browse/CIV-11304)

### Change description ###
Modified dashboard-notifications to :
- Send the params field when notifications are requested, to work in CUI with data included in the field
- Remove the notificationAction related to a notification that is updated to be displayed again.

Modified the "directions order (judgment)" trip to allow it to work on Lip v Lip or LR v Lip claims.

Added:
- Scripts, handlers, unit test and integration test to show the notification in the "final Order" "standard direction order" and "directions order (judgment)" trips.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x ] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/dashboardnotifications/defendant/OrderMadeDefendantNotificationHandler.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+    @Override
+    public List<CaseEvent> handledEvents() {
+        return EVENTS;
+    }
+
+    @Override
+    public String getScenario(CaseData caseData) {
+        return SCENARIO_AAA6_CP_ORDER_MADE_DEFENDANT.getScenario();
+    }
```
[the remark this answers is not in the document]
Comment (miguelMolina3691) [reply src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/dashboardnotifications/defendant/OrderMadeDefendantNotificationHandler.java:42]: That's right, added

File: src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/dashboardnotifications/claimant/OrderMadeClaimantNotificationHandler.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    @Override
+    public List<CaseEvent> handledEvents() {
+        return EVENTS;
+    }
+
+    @Override
+    public String getScenario(CaseData caseData) {
+        return SCENARIO_AAA6_CP_ORDER_MADE_CLAIMANT.getScenario();
+    }
+}
```
[the remark this answers is not in the document]
Comment (miguelMolina3691) [reply src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/dashboardnotifications/claimant/OrderMadeClaimantNotificationHandler.java:48]: That's right, added

**Review by laarmada — APPROVED** [review]

File: src/main/resources/db/migration/V2024_04_08_1220__CIV-11304-order_made_claimant.sql [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+                                                  ,notification_role, time_to_live)
+VALUES ('Notice.AAA6.CP.OrderMade.Claimant', 'An order has been made', 'An order has been made',
+        '<p class="govuk-body">The judge has made an order on your claim. <a href="{VIEW_FINAL_ORDER}" rel="noopener noreferrer" target="_blank" class="govuk-link">View the order</a>.</p>',
+        '<p class="govuk-body">The judge has made an order on your claim. <a href="{VIEW_FINAL_ORDER}" rel="noopener noreferrer" target="_blank" class="govuk-link">View the order</a>.</p>',
+        'CLAIMANT', 'Session');
+
+/**
+ * Add task list items
+ */
+INSERT INTO dbs.task_item_template (task_name_en, category_en, task_name_cy, category_cy, template_name,
```
Comment (kannan-v-hmcts) [inline src/main/resources/db/migration/V2024_04_08_1220__CIV-11304-order_made_claimant.sql:20]: @miguelMolina3691 Is this change needed ? If I am not wrong, in ur story, it says "View orders and notices - link is active, tag would already be Available"

**Review by laarmada — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/dashboardnotifications/claimant/OrderMadeClaimantNotificationHandler.java [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+    public String camundaActivityId(CallbackParams callbackParams) {
+        return TASK_ID;
+    }
+
+    @Override
+    public List<CaseEvent> handledEvents() {
+        return EVENTS;
+    }
+
+    public CallbackResponse configureDashboardScenario(CallbackParams callbackParams) {
```
Comment (dharmendrak) [inline src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/dashboardnotifications/claimant/OrderMadeClaimantNotificationHandler.java:49]: please add @Override here

Comment (miguelMolina3691) [reply src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/dashboardnotifications/claimant/OrderMadeClaimantNotificationHandler.java:50]: added

**Review by dharmendrak — DISMISSED** [review]

**Review by kannan-v-hmcts — DISMISSED** [review]

**Review by kannan-v-hmcts — DISMISSED** [review]

**Review by laarmada — APPROVED** [review]

**Review by kannan-v-hmcts — APPROVED** [review]

**Review by m-meulendijks-v1 — APPROVED** [review]
